src/main/java/Duke.java

```diff
@@ -1,10 +1,153 @@
+            + "Bye. Hope to see you again soon!\n"
+            + "____________________________________________________________\n";
+    public static final String WELCOME_MESSAGE = ""
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n"
+            + "____________________________________________________________\n"
+            + "Hello! I'm Duke\n"
+            + "What can I do for you?\n"
+            + "____________________________________________________________\n";
+
+    public static void printTaskMarkedDone (Task[] list, int checkedIntIndex){
+        System.out.println(""
+                + "____________________________________________________________\n"
+                + "Nice! I've marked this task as done:\n"
+                + "  " + list[checkedIntIndex - 1].getTaskIcon() + "["
+                + list[checkedIntIndex - 1].getStatusIcon() + "]"
+                + list[checkedIntIndex - 1].description
+                + "\n____________________________________________________________"
+        );
+    }
+
+    public static void printAddingTaskError(){
+        System.out.println(""
+                + "____________________________________________________________\n"
+                + "Please enter a valid input"
+                + "\n____________________________________________________________"
+        );
+    }
+
+    public static void printAddedTask(Task input, int index){
+        String task = "tasks";
+        if(index == 0){
+            task = "task";
+        }
+        System.out.println(""
+                + "____________________________________________________________\n"
+                + "Got it. I've added this task: \n"
+                + "  " + input.getTaskIcon() + "["
+                + input.getStatusIcon() + "]" + input.description
+                + "\nNow you have " + (index + 1) + " " + task + " in the list."
+                + "\n____________________________________________________________\n"
+        );
+    }
+
+    public static void printTaskList(Task[] list){
+        int count = 0;
+        System.out.println("____________________________________________________________");
+        while(list[count] != null){
+            System.out.println(""
+                    + (count + 1) + "." + list[count].getTaskIcon() + '['
+                    + list[count].getStatusIcon() + "]"
+                    + list[count].description
+            );
+            count ++;
+        }
+        System.out.println("____________________________________________________________");
+    }
+
+    //Processes
+    public static void addingTasksToList(){
+        Scanner in = new Scanner(System.in);
+        Task[] list = new Task[100];
+        int index = 0;
+        String input = in.nextLine();
+
+        while((!input.equals("bye")) && (!input.equals("Bye")) && (index < 100)){
```

Comment:
Consider using a case-insensitive comparison method as that can simplify the expression, a boolean variable to check for validity would also be good too.